tdms/include/unit_test_utils.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file unit_test_utils.h
+ * @brief Functions common to multiple unit tests
+ */
+#include <complex>
+
+template<typename T>
+/**
+ * @brief Determines if two numerical values are close by relative comparison.
+ *
+ * Checks the truth value of the condition
+ * abs(a - b) / max(abs(a), abs(b)) < tol.
+ * If a and b are both close to zero, returns true.
+ *
+ * @param a,b Numerical values
+ */
+inline bool is_close(T a, T b, double tol=1E-10, double close_to_zero_tol=1E-30) {
```

Comment:
```suggestion
/**
 * @brief Determines if two numerical values are close by relative comparison.
 *
 * Checks the truth value of the condition
 * abs(a - b) / max(abs(a), abs(b)) < tol.
 * If a and b are both close to zero, returns true.
 *
 * @param a,b Numerical values
 */
template<typename T>
inline bool is_close(T a, T b, double tol=1E-10, double close_to_zero_tol=1E-30) {
```

Check for me that this works? But I thought doxygen was smart enough to understand templating.